src/Symfony/Component/HttpKernel/CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ CHANGELOG
  * Add `#[MapQueryString]` to map and validate request query string from `Request::$query->all()` to typed objects
  * Add `#[MapQueryParameter]` to map and validate individual query parameters to controller arguments
  * Collect data from every event dispatcher
+ * Add optional `$className` parameter to `ControllerEvent::getAttributes()`
```

Comment:
This won't make it in 6.3 as we are 1.5 month after the feature freeze.

Reply:
Fixed